crates/walrus-simtest/tests/simtest_failure.rs

```diff
@@ -608,6 +608,7 @@ mod tests {
             .with_test_nodes_config(
                 TestNodesConfig::builder()
                     .with_node_weights(&[1, 2, 3, 3, 4])
+                    .with_node_recovery_config(node_recovery_config)
```

Comment:
This was causing a simtest build warning and was a regression caused by an earlier refactoring.